prac_08/convert_miles_km.kv

```diff
@@ -0,0 +1,30 @@
+BoxLayout:
+    orientation: 'vertical'
+    BoxLayout:
+        orientation: 'horizontal'
+        TextInput:
+            id: input_miles
+            text: '34'
+            font_size: 48
+            on_text: app.convert_miles_to_km()  # Update conversion in real-time
+        BoxLayout:
+            orientation: 'vertical'
+            size_hint_x: 0.25
+            Button:
+                text: 'Up'
+                on_press: app.handle_increment(1)
+            Button:
+                text: 'Down'
+                on_press: app.handle_increment(-1)
+    Button:
```

Comment:
I think we remove this for the final implementation, I just commented it out in mine.